java/debugger/impl/src/com/intellij/debugger/impl/JdiHelperClassLoaderImpl.kt

```diff
@@ -17,6 +17,10 @@ private class JdiHelperClassLoaderImpl : JdiHelperClassLoader {
     vararg additionalClassesToLoad: String,
   ): ClassType? {
     val vmProxy = evaluationContext.virtualMachineProxy
+    if (DexDebugFacility.isDex(vmProxy.virtualMachine)) {
```

Comment:
Can we introduce a separate method `isApplicable(evaluationContext: EvaluationContextImpl)` for that?